abstractions/python/kiota/abstractions/authentication/access_token_provider.py

```diff
@@ -0,0 +1,17 @@
+from abc import ABC, abstractmethod
+
+
+class AccessTokenProvider(ABC):
+    """Defines a contract for obtaining access tokens for a given url.
+    """
+    @abstractmethod
+    async def get_authorization_token(self, uri: str) -> str:
```

Comment:
in #1051 we added a getter to the allowed hosts validator, please add it here as well

Reply:
Thanks. Added https://github.com/microsoft/kiota/pull/1057/commits/a63fb7d2e76b2c6b180dc6184b7079a59913bf08